### DISCUSSION TITLE
from 2016 to 2022

### DISCUSSION THREAD

**Discussion by TomTravis** [dnschneid/crouton#4896] [Ideas]
sed and jupyterlab work great grep -rl was of great assistance now on to more release and installs 
I'm  in development full speed /usr/local mnt exec and dev
https://github.com/users/TomTravis/projects/3/views/1 join the fun
![Screenshot_2022-11-19_20-20-28](https://user-images.githubusercontent.com/46386261/202885455-32c6e51a-a245-489e-8f1b-70d0e37c9a97.png)
work great name what ever you wish
/var/host/media/removable/sd200

**Comment by the-d3fender** [comment]
Hi @TomTravis , seems like you're the only (or most), active developer, planning to make a brand-new crouton installation. So should I install the crouton from your repo or from this one, (Couldn't see any installation method from your repo btw) for the most stable and updated version?

**Comment by Penguinbot4** [comment]
this is a bot we still alive we busy creating code the link is for test2 same as crouton made changes for debootstrap to run in usr/local as requested by chromeOS
 Tom can no longer post here
![Screenshot_2022-11-27_13-53-14](https://user-images.githubusercontent.com/118845597/204161611-6feaaff4-f458-408e-89ba-d96bc8f114d9.png)
